test/e2e/test_helper.exs

```diff
@@ -17,6 +17,8 @@ Application.put_env(:phoenix_live_view, Phoenix.LiveViewTest.E2E.Endpoint,
   debug_errors: false
 )
 
+pid = self()
```

Comment:
`Process.register(self(), :e2e_helper)` and then send a message to the named process. :)